### ISSUE TITLE
🐛 UAT | Form labels repeating twice on windows

### ISSUE THREAD

**Issue by mauli0110** [GCTC-NTGC/gc-digital-talent#15900]
### 🐛 Bug and Expected Behaviour

The issue is same as https://github.com/GCTC-NTGC/gc-digital-talent/issues/15513 but happened on windows.

Expected Behaviour: The form label should announce only once

### 🕵️ Details

Env: UAT
OS: Windows

### 📋 Steps to Reproduce

Prerequisite: Make sure the Narrator started and it start announcing the page labels

1. Access to GC digital portal using an applicant role
2. Navigate to the new work experience form
3. Using narrator navigate to the form fields and verify the label is announcing twice. 
4. Navigate to personal information page and edit the work preferences 
5. Observe the Narrator is announcing the label twice

### 📸 Screenshot

https://github.com/user-attachments/assets/6fd90d2d-5bd7-4f9c-b348-b2cb2049ec5f

### ✅ Acceptance Criteria

- [ ] The Narrator should announced the label only once

### 🔄 Regression

No

**Comment by bella77-69** [comment]
I tested this on Windows using NVDA but was unable to reproduce the double announcement 🤷‍♀️ the label announced only once for me.

**Comment by mauli0110** [comment]
> I tested this on Windows using NVDA but was unable to reproduce the double announcement 🤷‍♀️ the label announced only once for me.

Did you try on UAT?

**Comment by bella77-69** [comment]
> > I tested this on Windows using NVDA but was unable to reproduce the double announcement 🤷‍♀️ the label announced only once for me.
> 
> Did you try on UAT?

Yes 👍
